### ISSUE TITLE
bug: ion-checkbox right not centered

### ISSUE THREAD

**Issue by ToKaDev** [driftyco/ionic#3900]
**Type**: <span ionic-type>bug</span>

**Platform**: <span ionic-platform>android</span> <span ionic-platform-version>5.0</span>  <span ionic-webview>webview</span>

<span ionic-description>Hi,
i use `<ion-checkbox class="item-checkbox-right">Administrators</ion-checkbox>` to create a right aligned checkbox. But in Android the 'check' is not align in the center of the box.

Screenshot:
![checkbox](https://cloud.githubusercontent.com/assets/7627815/8030362/923408c6-0dc6-11e5-941d-1b9dd212a91d.png)</span>

<span is-issue-template></span>

**Comment by Ionitron** [comment]
Greetings @ToKaDev!

My sensors indicate that you need to update your issue through our [custom issue form](http://ionicframework.com/submit-issue/#/?iid=3900). We are now requiring all issues to be submitted this way, to ensure that we have all of the information necessary to fix them as quickly as possible.

**[Click Here To Update Your Issue](http://ionicframework.com/submit-issue/#/?iid=3900)**

I will have no choice but to close this issue if it is not resubmitted through the form. Please fill out the rest of the form, so that I may use my friendly robot powers to assist you.

Thank you!

<span ionitron-issue-resubmit></span>

**Comment by sclearion** [comment]
Have you tried this markup? I tried yours and I see the same result you are talking about but the one below works fine
```html
<li class="item item-checkbox">
     <label class="checkbox">
       <input type="checkbox">
     </label>
     Chicken and Waffle Bubblegum
  </li>
```

**Comment by ToKaDev** [comment]
Hi,
i have tried your code sample, but it creates a checkbox on the left side of the list not on the right. I think there is a problem with the floating of the checkbox.

**Comment by ClemMakesApps** [comment]
Does this same issue also occur when running Ionic Labs? or just on device?

**Comment by ToKaDev** [comment]
It doesn't occur in Ionic Labs. Only on my Android emulator/device.

**Comment by ClemMakesApps** [comment]
This issue affects the android styling of Ionic regardless of whether its viewed in Ionic Labs, emulator or device.

Codepen: http://codepen.io/clemmakesapps/pen/WvOPmK

**Comment by mlynch** [comment]
Landed: #3933
